cross-chain-atomic-swap-cordapp/src/test/kotlin/jp/co/layerx/cordage/crosschainatomicswap/flow/LockEtherFlowTest.kt

```diff
@@ -32,12 +30,20 @@ class LockEtherFlowTest {
 
     @Before
     fun setUp() {
+
+        val customConfig: MutableMap<String, String> = LinkedHashMap()
+        customConfig["rpcUrl"] = "http://localhost:8545"
+        customConfig["networkId"] = "5777"
+        customConfig["privateKey"] = "0x6cbed15c793ce57650b9877cf6fa156fbef513c4e6134f022a85b1ffdd59b2a1"
```

Comment:
Do you think it is possible to read these data from config.conf that is placed in the `src/test/resources`?

Reply:
Done as well , added a 'readConfig' function in TestUtils.kt